tutorials/tour/_posts/2017-02-13-unified-types.md

```diff
@@ -11,15 +11,15 @@ next-page: classes
 previous-page: basics
 ---
 
-In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates the class hierarchy.
+In Scala, all values are instances of a class, including numerical values and functions. The diagram below illustrates a subset the class hierarchy.
```

Comment:
meh... I don't agree this is sufficient reason to leave it out of the diagram. the distinction between type constructors and types isn't important here. I wouldn't object to a footnote with a link to a later section where the issue you raise is addressed.